### PR TITLE
Run ESlint and Prettier on codebase

### LINKED ISSUE #6: Run ESlint and Prettier on codebase
currently: ESlint gives

✖ 54 problems (20 errors, 34 warnings)

### PR DESCRIPTION
**Pull request by MidasXIV**
## Description

| Files               | Refactoring type |
| :------------- | :---------------- |
| `.eslintrc.json` | make comments max length consistent with code max length |
| `src/components/app-header/index.tsx` | Add component types and Properties |
| `src/components/dropdown-menu/index.tsx` | Add component types and Properties |
| ` src/components/index.tsx` | file removed |
| `src/components/loading/index.tsx` | prettier fix |
| `src/components/menu/index.tsx` | Add component types and Properties |
| `src/pages/_app.tsx`  | Add component types and Properties |
| `src/pages/api/dividend.js` | import Dividend Controller as default module |
| `src/pages/api/dividendHistory.js` | import Dividend Controller as default module |
| `src/util/useOutsideClick.ts` | Add component types and Properties |
| `src/pages/dashboard.tsx` |  Add component types and Properties |
| `src/pages/index.tsx ` |  Add component types and Properties |
| `src/controllers/dividendController.ts` | run ESlint and prettier export class as default and make private functions static |

***

**Linked Issue**: https://github.com/MidasXIV/anavrin/issues/6

***

**Notes**

I still have the following issues, which I'll figure out within the next few days
```
  115:13  error    Expected the Promise rejection reason to be an Error   prefer-promise-reject-errors
  330:11  warning  Unexpected console statement                           no-console
  339:1   warning  This line has a length of 102. Maximum allowed is 100  max-len
  345:11  warning  Unexpected console statement                           no-console
  384:7   warning  Unexpected console statement                           no-console
  390:9   warning  Unexpected console statement                           no-console
```
✖ 6 problems (1 error, 5 warnings)

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import type { AppProps } from "next/app";
 import "../../public/global.css";
```
Comment (MidasXIV) [inline src/pages/_app.tsx:2]: why don't we move this to styles folder ?

**Comment by MidasXIV** [comment]
@kartiknair  Great work with the Refactoring

File: src/pages/_app.tsx (hunk 1 above)
Comment (kartiknair) [reply src/pages/_app.tsx:2]: That's a good idea. I think we can do that in the same PR where we migrate from styled-jsx to CSS modules

**Review by kartiknair — APPROVED** [review]
LGTM. Tested out the API & it's working as expected

**Comment by MidasXIV** [comment]
Hey, if you waiting on the merge for this one. You can go ahead and merge it in. Thanks for cross checking the API.

**Pull request merged**